mobile/test/cc/integration/lifetimes_test.cc

```diff
@@ -64,20 +19,18 @@ struct Status {
 void sendRequest(Platform::EngineSharedPtr engine, Status& status,
                  absl::Notification& stream_complete) {
   auto stream_prototype = engine->streamClient()->newStreamPrototype();
-  auto stream = (*stream_prototype)
-                    .setOnHeaders([&](Platform::ResponseHeadersSharedPtr headers, bool end_stream,
-                                      envoy_stream_intel) {
+  auto stream = stream_prototype
+                    ->setOnHeaders([&](Platform::ResponseHeadersSharedPtr headers, bool end_stream,
```

Comment:
nit: Slightly preference for the existing style because all of the parallel setFoo() builder methods use "." so it's visually more consistent.

Reply:
Done.